### DISCUSSION TITLE
Disabling Admin Console UI

### DISCUSSION THREAD

**Discussion by leorl1412** [keycloak/keycloak#20583] [Q&A]
Hello,

I am trying to optimize my Keycloak Container and I want to know if there is a way to disable Keycloak's Admin Console? Because I will not be using it at all.

Thanks,

**Comment by jonkoops** [comment]
If you are looking to disable specific features of Keycloak see the ["Enabling and disabling features" section](https://www.keycloak.org/server/features) of the documentation. In your case you are looking to disable the `admin2` feature (the second iteration of the Admin Console), the command will look something like:

```sh
bin/kc.[sh|bat] build --features-disabled="admin2"
```

**Reply by pravinjec** [reply]
i have ran this command and still admin console is accessible
